### DISCUSSION TITLE
能否让益达的视频播放器能支持磁力在线播放

### DISCUSSION THREAD

**Discussion by dabaiyei** [xiaohucode/yidaRule#148] [Ideas]
能否让益达的视频播放器能支持磁力在线播放

**Comment by xiaohucode** [comment]
1.1.24 版本已支持

**Reply by a1850976305** [reply]
大佬1.1.24版本是不是还没放出来？找不到安装包

**Comment by dabaiyei** [comment]
大佬牛掰

**Comment by mr-dragon** [comment]
> 1.1.24 版本已支持

怎么 release 这个版本没有文件？求大佬更新

**Comment by hyhwl** [comment]
![image](https://github.com/xiaohucode/yidaRule/assets/15354279/d43ca75f-92ab-420a-8bae-4026422cee92)
初始化失败

**Reply by xiaohucode** [reply]
解压。

**Reply by hyhwl** [reply]
解压了才能打开程序。你说的解压 是指什么操作

**Comment by fuxin-mail** [comment]
![image](https://github.com/xiaohucode/yidaRule/assets/64723923/93f0e43b-e19a-4e70-b4d7-e5e94a4fd6b2)
磁力解析下面的Tracker服务器咋填？输入磁力链接按钮是灰色不可点呢？

**Reply by xiaohucode** [reply]
灰色是因为服务启动失败，有提示的

**Reply by SONGjiemo** [reply]
mac 版本输入磁力后，没有任何的反应。就是点击解析后  30S 就结束了也不说有没有成功

**Comment by fuxin-mail** [comment]
> 灰色是因为服务启动失败，有提示的

服务器启动失败咋解决？
